pages/1.13/administering-clusters/locate-public-agent/index.md

```diff
@@ -6,23 +6,193 @@ menuWeight: 3
+    ```
+
+    In this example, there are three IP addresses for the public agents:
+    - 34.212.37.79
+    - 52.25.254.97
+    - 54.218.23.75
+
+    There are no public IP addresses available for the private nodes. 
+
+## Looking up addresses from a master node
+You can also find the public IP addresses by using the client URL (`cURL`) command from a master node. For example, you can find the public-facing IP addresses for public agents using the following cURL command: 
+
+```bash
+curl -skSL -H "Authorization: token=$(dcos config show core.dcos_acs_token)" -H 'Content-Type: application/json' $(dcos config show core.dcos_url)/net/v1/nodes
+```
+
+This command returns output similar to the following: 
+
+```bash
+[{"updated":"2019-01-07T22:22:22.171Z","public_ips":["34.212.37.79"],"private_ip":"10.0.6.210","hostname":"ip-10-0-6-210"},{"updated":"2019-01-07T22:22:22.119Z","public_ips":["52.25.254.97"],"private_ip":"10.0.6.181","hostname":"ip-10-0-6-181"},{"updated":"2019-01-07T22:21:09.585Z","public_ips":["54.218.23.75"],"private_ip":"10.0.6.148","hostname":"ip-10-0-6-148"},{"updated":"2019-01-07T22:22:28.582Z","public_ips":[],"private_ip":"10.0.1.139","hostname":"ip-10-0-1-139"},{"updated":"2019-01-07T22:22:28.649Z","public_ips":[],"private_ip":"10.0.0.138","hostname":"ip-10-0-0-138"}]
+```
+
+You can also execute the cURL command directly on a master node find the public IP addresses for public agents from a central location: 
+
+```bash
+curl http://localhost:62080/v1/nodes
+```
+
+This command output displays the public IP addresses of the same public agents that are running on the DC/OS cluster. For example: 
+
+    ```bash
+    curl http://localhost:62080/v1/nodes
+    [{"updated":"2019-01-07T22:22:22.171Z","public_ips":["34.212.37.79"],"private_ip":"10.0.6.210","hostname":"ip-10-0-6-210"},{"updated":"2019-01-07T22:22:22.119Z","public_ips":["52.25.254.97"],"private_ip":"10.0.6.181","hostname":"ip-10-0-6-181"},{"updated":"2019-01-07T22:21:09.585Z","public_ips":["54.218.23.75"],"private_ip":"10.0.6.148","hostname":"ip-10-0-6-148"},{"updated":"2019-01-07T22:22:28.582Z","public_ips":[],"private_ip":"10.0.1.139","hostname":"ip-10-0-1-139"},{"updated":"2019-01-07T22:22:28.649Z","public_ips":[],"private_ip":"10.0.0.138","hostname":"ip-10-0-0-138"}]
+    ```
+
+    In both of these examples, there are three public IP addresses, one for each of the public agent nodes:
+    - 34.212.37.79
+    - 52.25.254.97
+    - 54.218.23.75
+
+    As with the previous example, there are no public IP addresses available for the private nodes. 
+
+## Formatting API output using jq
```

Comment:
In concert with my above note, perhaps move this earlier in the doc, show both an formatted and formatted response and make the note that from here on out all responses have been formatted for readability.

Reply:
great suggestion.